external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaSparkConfSuite.scala

```diff
@@ -1,30 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.spark.sql.kafka010
-
-import org.apache.spark.{LocalSparkContext, SparkConf, SparkFunSuite}
-import org.apache.spark.util.ResetSystemProperties
-
-class KafkaSparkConfSuite extends SparkFunSuite with LocalSparkContext with ResetSystemProperties {
```

Comment:
Hmm, what has happened with this test?

Reply:
I have renamed the config to old one per feedback, so no longer need this test.

Reply:
Which comment are you referring to? Looking for it but maybe closed.

Reply:
Sigh, I guess I messed thing up while reflecting review comments. Nice finding. These things are not related to this patch. Just rolled back.

Reply:
Thanks, I think it can be resolved.